scripts/util/deploy-protocol-client-proxies.js

```diff
@@ -19,18 +20,18 @@ const confirmations = environments.confirmations;
  *
  * @param protocolClients
  * @param protocolClientArgs
- * @param gasLimit - gasLimit for transactions
+ * @param maxPriorityFeePerGas - maxPriorityFeePerGas for transactions
  * @returns {Promise<(*|*|*)[]>}
  */
-async function deployProtocolClientProxies(protocolClients, gasLimit) {
+async function deployProtocolClientProxies(protocolClients, maxPriorityFeePerGas) {
   let bosonClientBeacon;
 
   // Destructure the protocol client implementations
   [bosonClientBeacon] = protocolClients;
 
   // Deploy the ClientProxy for BosonVoucher
   const ClientProxy = await ethers.getContractFactory("BeaconClientProxy");
-  const clientProxy = await ClientProxy.deploy({ gasLimit });
+  const clientProxy = await ClientProxy.deploy(await getFees(maxPriorityFeePerGas));
   await clientProxy.deployTransaction.wait(confirmations);
 
   // init instead of constructors
```

Comment:
Do need this for the initialize call just below this on line 38?

Reply:
Added.